system/jlib/jtrace.hpp

```diff
@@ -66,6 +66,7 @@ interface ISpan : extends IInterface
     virtual void getClientHeaders(IProperties * clientHeaders) const = 0;
     virtual void toString(StringBuffer & out) const = 0;
     virtual void getLogPrefix(StringBuffer & out) const = 0;
+    virtual bool isRecording() const = 0;   // Is it worth adding any events/attributes to this span?
```

Comment:
I think this is a rhetorical question, but no events/attributes can be added if not recording, but some ISpan pass-through values are "recorded" and exported